src/renderer/content.ts

```diff
@@ -7,7 +7,8 @@ import { readFiddle } from '../utils/read-fiddle';
 import * as fsType from 'fs-extra';
 import * as path from 'path';
 import * as semver from 'semver';
-import * as decompress from 'decompress';
+
+const decompress = require('decompress');
```

Comment:
nit: does `import =` work instead?

```suggestion
import decompress = require('decompress');
```

Reply:
This is moot now that ba8a22f40e0a3900078112c4aab154547e2634ba is merged. This will do the Right Thing after rebasing.

Reply:
This is resolved now